src/wp-admin/includes/theme.php

```diff
@@ -1224,11 +1224,14 @@ function paused_themes_notice() {
 		return;
 	}
 
-	printf(
-		'<div class="notice notice-error"><p><strong>%s</strong><br>%s</p><p><a href="%s">%s</a></p></div>',
-		__( 'One or more themes failed to load properly.' ),
-		__( 'You can find more details and make changes on the Themes screen.' ),
-		esc_url( admin_url( 'themes.php' ) ),
-		__( 'Go to the Themes screen' )
+	wp_admin_notice(
+		sprintf(
+			'<strong>%s</strong><br>%s</p><p><a href="%s">%s</a>',
```

Comment:
I think that having partial `p` elements nested inside the wrapper makes the code less readable; should probably include the wrapping `p` elements and add `paragraph_wrap` flag.